src/test/resources/maven-bench.properties

```diff
@@ -23,7 +26,7 @@ warmup.number = 1
 measures.number-by-maven-version = 10
 
 # Path where measures will be exported
-measures.export.path = C:\\Maven\\measures
+measures.export.path = ${project.build.testOutputDirectory}/maven/measures
```

Comment:
Actually in case of development, I strongly suggest to move out of build directory :+1: 
I think I put it on README.md on "Custom Build" section under "Contributing" part.